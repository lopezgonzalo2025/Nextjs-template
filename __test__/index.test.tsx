import { render } from '@testing-library/react'

import Home from '../src/app/page'

describe('<Index />', () => {
  let component: any

  beforeEach(() => {
    component = render(<Home />)
  })

  test('Snapshots', () => {
    const { container } = component
    expect(container).toMatchSnapshot()
  })

  test('Render Home', () => {
    const textsToSearch = ['Home']

    textsToSearch.forEach((text) => {
      component.getByText(text)
    })
  })
})
