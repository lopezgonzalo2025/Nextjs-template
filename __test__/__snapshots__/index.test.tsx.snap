// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> Snapshots 1`] = `
<div>
  <main>
    Home
  </main>
</div>
`;
